src/QsCompiler/QirGeneration/EntryPointOperationLoader.cs

```diff
@@ -0,0 +1,70 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using Microsoft.Quantum.Qir.Serialization;
+using Microsoft.Quantum.QsCompiler.SyntaxTokens;
+using Microsoft.Quantum.QsCompiler.SyntaxTree;
+
+namespace Microsoft.Quantum.QsCompiler.QIR
+{
+    using QsTypeKind = QsTypeKind<ResolvedType, UserDefinedType, QsTypeParameter, CallableInformation>;
+
+    public static class EntryPointOperationLoader
+    {
+        public static IList<EntryPointOperation> LoadEntryPointOperations(FileInfo assemblyFileInfo)
```

Comment:
Can you please add a documentation comment to this method since it's public?

Reply:
added